cloudpickle/cloudpickle_fast.py

```diff
@@ -537,6 +525,8 @@ def _reduce_global(pickler, obj):
     we return a reduce value the the Pickler will internally serialize via
     save_reduce.
     """
+    # XXX: This functions needs the current active pickler to pass its
```

Comment:
function

Reply:
why is this an XXX comment? If this is just an informative comment with no changes planned, please remove the XXX.

Reply:
Anytime I labeled a comment with `XXX`, I supposed you might have an opinion on the implementation related to the comment.
Regarding this specific comment, I think it is unusual for a reducer to be stateful (it is stateful here because it takes the current pickler as an argument) - I was simply waiting for validation on your end.

Reply:
I think it's fine. BTW technically `_reduce_global` is not a regular reducer but a hook :)

Reply:
Maybe we could rename it to `_reducer_override` here also to make that more explicit.